src/test/java/uk/ac/cam/cl/dtg/isaac/api/AuthenticationFacadeIT.java

```diff
@@ -1,11 +1,11 @@
 package uk.ac.cam.cl.dtg.isaac.api;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import static org.eclipse.jetty.http.HttpCookie.SAME_SITE_LAX_COMMENT;
 import static org.junit.Assert.assertEquals;
```

Comment:
Since you're extending from Jupiter now, you may as well change this.

In fact, could you please create a new ticket to replace all the usages of Junit4 with their Junit5 equivalent? Providing we don't have one already.